app/models/uittreksel.js

```diff
@@ -8,9 +8,10 @@ export default class UittrekselModel extends Model {
   type = 'http://data.lblod.info/id/document-types/uittreksel';
 
   rdfaBindings = {
-    class: 'foaf:Document',
+    class:
+      'foaf:Document https://data.vlaanderen.be/id/concept/BesluitDocumentType/9d5bfaca-bbf2-49dd-a830-769f91a6377b',
```

Comment:
I'm not very familiar with how this rdfaBindings config works, but this looks a bit weird. Just making sure this was an intended change :smile:.

Reply:
Apparently yes. Giving multiple types is like giving multiple classes on a HTML element; they're just separated with a space. Checked with an RDFa parser.